sixtyfps_runtime/rendering_backends/gl/graphics_window.rs

```diff
@@ -557,6 +557,65 @@ impl PlatformWindow for GraphicsWindow {
         result
     }
 
+    fn text_input_position_for_byte_offset(
+        &self,
+        text_input: Pin<&corelib::items::TextInput>,
+        byte_offset: usize,
+    ) -> Point {
+        let scale_factor = self.scale_factor();
+        let text = text_input.text();
+
+        let mut result = Point::default();
+
+        let width = text_input.width() * scale_factor;
+        let height = text_input.height() * scale_factor;
+        if width <= 0. || height <= 0. {
+            return result;
+        }
+
+        let font = text_input
```

Comment:
Also unrelated, but this is starting to look like an opportunity for moving the code into a shared function for determining the font.